espnet2/tasks/asr.py

```diff
@@ -369,6 +377,9 @@ def build_preprocess_fn(
                 speech_volume_normalize=args.speech_volume_normalize
                 if hasattr(args, "rir_scp")
                 else None,
+                aux_task_names=cls.optional_data_names(
+                    train=train, aux_tasks=args.aux_ctc_tasks
+                ),
```

Comment:
The CI does not pass because previous checkpoints does not have the fields. To make it downwards-compatible, please use the similar tricks as the noise augmentation augments (by using `if hasattr`).